blog-homepage/src/cards/Card.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Audio from "../component/Audio";
+import PropTypes from "prop-types";
+import classes from "./Card.module.css";
+import bookmarkIcon from "../icons/bookmark.svg";
+import bookmarkIconGreen from "../icons/bookmarkGreen.svg";
+
+function Card({
+  articleLength,
+  audio,
+  author,
+  authorImage,
+  isCardMain,
+  date,
+  description,
+  image,
+  website,
+  memberPreview,
+  title,
+}) {
+  const [isBookmarked, setIsBookmarked] = useState(false);
+
+  const bookmarkHandler = () => {
+    setIsBookmarked((previousState) => !previousState);
+  };
+
+  const openSiteInTab = (url) => {
+    window.open(url, "_blank");
+  };
+
+  return (
+    <div className={isCardMain ? classes.card : classes["card-alt"]}>
+      <img
+        className={isCardMain ? "" : classes["img-alt"]}
+        src={image}
+        alt="main article jpeg"
+        onClick={() => openSiteInTab(website)}
+      />
+      {/* {Create grid. Grid format is based on card rendering in Main or Lower section.} */}
+      <div className={isCardMain ? classes.content : classes["content-alt"]}>
+        <div className={isCardMain ? classes.main : classes["main-alt"]}>
+          {isCardMain && audio && <Audio member={memberPreview} />}
+          {!isCardMain && memberPreview && (
+            <div className={classes.star}>★ Member Preview</div>
+          )}
+          <h2 onClick={() => openSiteInTab(website)}>{title}</h2>
+          <p onClick={() => openSiteInTab(website)}>{description}</p>
+        </div>
+        <div className={isCardMain ? classes.avatar : classes["avatar-alt"]}>
```

Comment:
these multiple checks for if it's isCardMain might get really hard to keep track of. It's out of the scope of this class, but scss would be helpful for this. But I'd argue that since you have to do a lot of modifying of css it might just be easier to make two components rather than one larger one that requires so much conditionally rendering.

Reply:
I did start this homework with seperate components to render the main and lower section and then attempted to make a reusable component and ended with this. Thank for letting me know about scss, I will take it look into it.

Reply:
SCSS is pretty cool, and if you're decent at CSS it's not that hard to pick up. and you can just import scss files directly into react components now, which makes life so much easier.